src/envoy/http/jwt_auth/http_filter_factory.cc

```diff
@@ -28,7 +30,7 @@ class JwtVerificationFilterConfig : public NamedHttpFilterConfigFactory {
   HttpFilterFactoryCb createFilterFactory(const Json::Object& config,
                                           const std::string&,
                                           FactoryContext& context) override {
-    Http::JwtAuth::Config::AuthFilterConfig proto_config;
+    JwtAuthentication proto_config;
```

Comment:
Can we support both proto format? I.e move the ConvertProto from mixer to here, and try to parse json config to both proto type (only one would success, I guess)

It would be safer as we will need to sync with pilot changes (code change + proxy SHA update)? If it cause too much trouble, then we can plan to submit all PRs together.

Reply:
EndUserSpec is only used by Mixer.  Unless you want Pilot to use EndUserSpec to pass filter config.
Beside, I don't want this module to deal with EndUserSpec.

Reply:
The new authn policy already insert jwt filter (not the one under mixer) using EUSpec proto. While it's still in private testing, I try to not breaking it neither. The least we should do is prepare changes in both sides and try to submit together (I'll do it today for pilot change anyway).

With respect to EUC in mixer, after this changes, I think we should start removing it (proto and pilot) to steer user to use authn policy.